src/sentry/testutils/helpers/features.py

```diff
@@ -7,7 +7,21 @@
 
 
 @contextmanager
-def Feature(name, active=True):
+def Feature(names):
+    """
+    Control whether a feature is enabled.
+
+    >> with Feature({'feature-1': True, 'feature-2': False}):
+    >>   # Executes with feature-1 enabled and feature-2 disabled
+
+    The following two invocations are equivalent:
+
+    >> with Feature(['feature-1', 'feature-2']):
+    >> with Feature({'feature-1': True, 'feature-2': True}):
+    """
+    if isinstance(names, list):
```

Comment:
It might be safer to invert this logic and go with `collections.Mapping` so that other sequence types (tuple, set) can still be used rather than restricting it to lists. That could have an edge case if a single string is provided (and isn't wrapped in another sequence type) that you'd have to address

Reply:
In that case the dict is included in the type check and we'll lose the flag:

```
>>> names = {'feature1': False, 'feature-2': True}
>>> names = {k: True for k in names}
{'feature1': True, 'feature-2': True}
```

Reply:
I don't understand this reply, can you clarify/elaborate

Reply:
Ah sorry no I'm being silly. Ignore me

I misunderstood what you meant by inverting the logic (also my example is wrong). You're saying invert like this right:

```
if not isinstance(names, collections.Mapping):
```

Reply:
Yep — sorry, I probably should have been more specific

Reply:
b6c523e — I don't know what a steerable object is or why I typed that, but it's my birthday so it's staying like that until it's squashed.